test/equipment.test.js

```diff
@@ -0,0 +1,12 @@
+import Equipment from '../js/equipment';
+
+describe('Equipment', () => {
+    describe('class', () => {
```

Comment:
Lets stick with some style guide, i think this days tabs is not a good option to indent code

Reply:
It's not a tab, it 4 spaces :) Fixed it already.